### DISCUSSION TITLE
We cannot open "Pull Request" tab from github web mobile

### DISCUSSION THREAD

**Discussion by anatawa12** [community/community#77823] [Mobile] [answered]
### Select Topic Area

Bug

### Body

We cannot open "Pull Request" tab from github web mobile.

![IMG_3379](https://github.com/community/community/assets/22656849/841ccef8-79c5-47c6-adc9-27715c89bd21)

**Comment by dakshithadissanayaka** [comment]
You are correct. Currently, there is no way to open the "Pull Request" tab from the GitHub web mobile app. However, you can still view and manage pull requests from the web app by using the following workarounds:

Use the desktop app or website: If you need to create or edit a pull request, you can use the GitHub desktop app or the GitHub website on a desktop computer.

Use the "Issues" tab: You can view a list of all open pull requests for a repository by going to the "Issues" tab and clicking on the "Pull requests" filter.

Use the search bar: You can search for pull requests by name or by using keywords. For example, to search for all pull requests that mention "bug fix," you would type "bug fix" into the search bar.

Use the API: You can use the GitHub API to view, create, and edit pull requests.

I hope this helps!

**Reply by anatawa12** [reply]
I'm using manually entering URL to see Pull Requests as a workaround. Thank you for suggesting solution

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @anatawa12! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by anatawa12** [reply]
Thank you for fixing category!

**Comment by anatawa12** [comment, marked as answer]
Recent update to GitHub UI fixes this problem!
